packages/_vue3-migration-test/src/components/animations/test-animate-width.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <button @click="visible = !visible">Toggle</button>
+    <AnimateWidth>
+      <div v-if="visible" style="width: 300px">Element to animate</div>
+    </AnimateWidth>
+  </div>
+</template>
+
+<script lang="ts">
```

Comment:
You can use `script setup` here if you want
```
<script setup lang="ts">
  import { ref } from 'vue';
  import AnimateWidth from '../../../../x-components/src/components/animations/animate-width.vue';

  const visible = ref(true);
</script>
```

Also changing it, this TS error is solved
<img width="760" alt="Screenshot 2024-04-18 at 6 37 33 PM" src="https://github.com/empathyco/x/assets/10746604/d78be8d0-7615-4271-aaf4-8b85a5dd884a">